test/pkg/environment/aws/environment.go

```diff
@@ -64,7 +70,16 @@ func NewEnvironment(t *testing.T) *Environment {
 		EC2API:          ec2.New(session),
 		SSMAPI:          ssm.New(session),
 		IAMAPI:          iam.New(session),
+		CloudwatchAPI:   GetCloudWatchAPI(session),
 		InterruptionAPI: itn.New(lo.Must(config.LoadDefaultConfig(env.Context))),
 		SQSProvider:     interruption.NewSQSProvider(sqs.New(session)),
 	}
 }
+
+func GetCloudWatchAPI(session *session.Session) cloudwatchiface.CloudWatchAPI {
+	if lo.Must(env.GetBool("ENABLE_CLOUDWATCH", false)) {
```

Comment:
just inferring, but shouldn't this be the following?
```suggestion
	if lo.Must(env.GetBool("ENABLE_CLOUDWATCH", true)) {
```

Reply:
No, we want to disable this by default, unless someone explicitly overrides it.

Reply:
My bad here, misunderstood what the env.GetBool() second parameter meant.